seirmo/figures.py

```diff
@@ -0,0 +1,99 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import plotly.graph_objects as go
+
+
+class IncidenceNumberPlot(object):
+    """
+
+    Creates incidence number plots for the app
+
+    """
+    def __init__(self):
+        super(IncidenceNumberPlot, self).__init__()
+
+        # Instantiate an empty plotly figure
+        self._fig = go.Figure()
+
+        """
+        Add a bar plot trace for the given incidence number data
+        in the figure
+
+        Parameters
+        ----------
+        data: pandas.DataFrame
+        time_key: docstring
+        inc_key: docstring
+        """
+    def add_data(self, data, time_key='Time',
+                 inc_key='Incidence Number'):
+
+        # Plot a bar chart for the data
+        self._fig.add_trace(
+            go.Bar(
+                x=data[time_key],
+                y=data[inc_key],
```

Comment:
For the bar plot it probably makes sense to bin the data not automatically, but to have one bar for each basic time unit step. If you look at the width argument in the documentation https://plotly.com/python/reference/bar/ it's not very clear what the default behaviour is, but it's probably safer to set `width=1`.